bigtable_rs/src/bigtable.rs

```diff
@@ -384,7 +401,9 @@ fn create_client(
     } else {
         "https://www.googleapis.com/auth/bigtable.data".to_string()
     };
-    let channel = Channel::balance_list(endpoints.into_iter());
+
+    let channel = channel.unwrap_or_else(|| Channel::balance_list(endpoints.into_iter()));
```

Comment:
Perhaps we can simplify further by just letting `fn create_client` take in a single Channel?

Then `Channel::balance_list(endpoints.into_iter())` can be called in other usual code path before the call to `fn create_client(channel ...)`

Reply:
SGTM! I'll update the PR in the next few days.